sandbox/monitoring/src/main/java/brooklyn/entity/monitoring/zabbix/ZabbixFeed.java

```diff
@@ -305,6 +305,7 @@ protected ZabbixFeed(final Builder<? extends ZabbixFeed, ?> builder) {
 
         Set<ZabbixPollConfig<?>> polls = Sets.newLinkedHashSet();
         for (ZabbixPollConfig<?> config : builder.polls) {
+            if (!config.isEnabled()) continue;
```

Comment:
This seems like the sort of thing that's easy to forget. Worth adding `builder.enabledPolls`?